schemas/responses/progressResult.yaml

```diff
@@ -0,0 +1,8 @@
+ProgressResult:
```

Comment:
nit: I feel we can merge the `response` with `endpoint`  so that `endpoint.yaml` contains more complete information of each endpoint.  for `progressResult` we can move it to `component` folder since it is shared by multiple endpoints( also it be merged with `stepProgress.yaml`.